map_list/map_list.tsx

```diff
@@ -17,6 +18,31 @@ type Props = {
   onChangeFilterQuery(val: string): void,
 };
 
+function getComplexityColor(complexity: number) {
+  switch (complexity) {
+    case 1:
+      return 'green';
+    case 2:
+      return 'yellow';
+    case 3:
+      return 'orange';
+    case 4:
+      return 'red';
+    case 5:
+      return 'black';
+    default:
+      throw new Error(`Did not expect complexity level ${complexity}`);
+  }
+}
+
+export const ComplexityColorPills = (props: { complexities: Complexity[] }) => (
+  <div className={styles.complexities}>
+    {props.complexities.map((c, i) => (
+        <div key={i} className={styles.complexityColorPill} style={{ backgroundColor: getComplexityColor(c.complexity)}}></div>
```

Comment:
I'll be adding dprint support in the next PR, so not worrying about formatting too much at this point.